### PR TITLE
ADR: PII Management Strategy

### PR DESCRIPTION
**Pull request by ttoomey**
## _Merging this PR_
- use the squash-merge strategy

## Description

Proposed PII Management Strategy
[reference #6730](https://github.com/open-path/Green-River/issues/6730)

## How to review
Our ADRs process is brand-new. [See ADR process doc](https://github.com/greenriver/hmis-warehouse/blob/release-142/docs/adr/README.md)

## Type of change
ADR proposal

### CODE REVIEW

**Comment by ttoomey** [comment]
This is a draft ADR, feedback is welcome. Please feel free to edit the ADR directly.

**Review by eanders — APPROVED** [review]

**Review by dtgreiner — APPROVED** [review]

**Review by martha — APPROVED** [review]

File: docs/adr/0002-pii-management-strategy.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+The phased approach allows us to:
+
+1. Start with low-risk tracking improvements
+2. Gather data to inform protection strategies
+3. Incrementally enhance security measures
+4. Maintain system stability throughout changes
+
+### Related Documents
+
+- [#6730](https://github.com/open-path/Green-River/issues/6730)
```
Comment (gigxz) [inline docs/adr/0002-pii-management-strategy.md]: This ticket feels only marginally related. I would think the work you did around the pii access class is equally relevant. Do we need some new issues or maybe an Epic to track these 3 phases outlined?

File: docs/adr/0002-pii-management-strategy.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+- Improve PII access control implementation for common reports and dashboards
+- Implement systemic PII tracking mechanism in application code
+- Define sensitivity levels and other metadata for PII fields
+- Create inventory of data models containing PII
+- Investigate PII requirements in our BI tooling
+
+### Phase 2: Enhanced Tooling & Management
+
+- Implemnent systemic PII access control across all locations that expose level 1 PII (dashboards and reports)
```
Comment (gigxz) [inline docs/adr/0002-pii-management-strategy.md]:
```suggestion
- Implement systemic PII access control across all locations that expose level 1 PII (dashboards and reports)
```

File: docs/adr/0002-pii-management-strategy.md (hunk 1 above)
Comment (gigxz) [reply docs/adr/0002-pii-management-strategy.md]: Although nvm you did go above and beyond on inventorying PII for that clearout ticket, so the linked PR is relevant

**Review by gigxz — APPROVED** [review]

File: docs/adr/0002-pii-management-strategy.md (hunk 1 above)
Comment (ttoomey) [reply docs/adr/0002-pii-management-strategy.md]: good idea, I made a placeholder epic for reference

**Review by eanders — APPROVED** [review]

**Review by gigxz — APPROVED** [review]

**Pull request merged**